### DISCUSSION TITLE
Bundles related to Transplants

### DISCUSSION THREAD

**Discussion by Ginger228** [CMSgov/price-transparency-guide#85] [Q&A] [answered]
What is the recommendation for negotiated rates related to facilities that are institutes of excellence for transplants. The facility has the negotiated rate for all services and pays the providers directly.  The burden doesn't fall on the payor. 

Should all codes (anesthesia, x-rays, surgery, hospital stay, etc.)  be listed with the facility?

**Comment by shaselton-usds** [comment, marked as answer]
@Ginger228 Trying to wrap up old threads here.

If the facility has directly negotiated with the provider then it wouldn't fall under the requirements of the rule. Reporting is required where negotiations are between the payer and facility/provider for services/items. Just double checking that this is your question.

**Reply by Ginger228** [reply]
Yes- this does answer the question.  The rate to display is between the payer and facility.  Any payments the facility makes with the providers are out of scope. Thanks.
